quicksort/c/quicksort.c

```diff
@@ -0,0 +1,49 @@
+/*
+Quicksort is a divide and conquer algorithm. The steps are:
+1) Pick an element from the array, this element is called as pivot element.
+2) Divide the unsorted array of elements in two arrays with values less than the pivot come in the first sub array, while all elements with values greater than the pivot come in the second sub-array (equal values can go either way). This step is called the partition operation.
+3) Recursively repeat the step 2(until the sub-arrays are sorted) to the sub-array of elements with smaller values and separately to the sub-array of elements with greater values.
+The same logic I have implemented in the following C program.
+*/
+
+#include<stdio.h>
+void quicksort(int number[25], int first, int last) {
+	int i, j, pivot, temp;
+
```

Comment:
Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]

*Origin: CPPLintBear, Section: `all.cpplint`.*